CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## 56 Release Notes
 
+### Retry with More Memory as workflow option
+
+The experimental memory retry feature gains per-workflow customization and includes breaking changes:
```

Comment:
TOL is it still "experimental"? Should we remember to mark it as such in the UI?

Reply:
IMO it's experimental for Cromwell customers because we are still making breaking changes.

For Terra customers, we are handling those changes within Terra so it's not experimental.

Reply:
I feel this is still experimental since it is only supported in Papiv2. Do you have a strong preference for removing it?